src/components/Card.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+	<element :is="isLink ? 'a' : 'div'"
+		:href="href"
```

Comment:
Otherwise `div` has empty invalid `html` attribute.
```suggestion
	<element :is="isLink ? 'a' : 'div'"
		:href="href || undefined"
```

Reply:
Or in the same style as attrs bellow:

```suggestion
	<element :is="isLink ? 'a' : 'div'"
		:href="!isLink ? href : "
```